.github/workflows/tag.yml

```diff
@@ -23,3 +23,5 @@ jobs:
         uses: goreleaser/goreleaser-action@v1
```

Comment:
not related to this PR, but we can also bump to v2 for this action as well.

Reply:
Should I change it? it will take 2 minutes

Reply:
yes please :)

Reply:
ok